release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -364,6 +364,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-networking"]
 === Networking
 
+[id="ocp-4-15-ipv6-gratuitous-arps-default-macvlan"]
+==== IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on macvlan CNI plugin
+
+Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/NDP caches with the correct information.
```

Comment:
Just let you know, according to https://issues.redhat.com/browse/NP-807, QE only test IPv6 but not IPv4